molo/profiles/admin.py

```diff
@@ -110,3 +127,116 @@ class FrontendUsersModelAdmin(WagtailModelAdmin, ProfileUserAdmin):
 
     def get_queryset(self, request):
         return User.objects.filter(profile__site=request.site)
+
+
+class TzDateTimeWidget(DateTimeWidget):
+
+    def render(self, value, obj):
+        if settings.USE_TZ:
+            value = localtime(value)
+        return super(TzDateTimeWidget, self).render(value, obj)
+
+
+class MergedCMSUserResource(ModelResource):
+    date_of_birth = Field(
+        'profile__date_of_birth', 'date_of_birth', widget=DateWidget())
+    alias = Field('profile__alias', 'alias')
+    mobile_number = Field('profile__mobile_number', 'mobile_number')
+    gender = Field('profile__gender', 'gender')
+    migrated_username = Field(
+        'profile__migrated_username', 'migrated_username')
+    security_question_answers = Field(
+        'profile__security_question_answers',
+        'security_question_answers')
+    date_joined = Field(
+        'date_joined', 'date_joined', widget=TzDateTimeWidget())
+    site = Field('profile__site__pk', 'site')
+
+    class Meta:
+        model = User
+        exclude = ('id', 'is_superuser', 'groups',
+                   'user_permissions', 'is_staff', 'last_login')
+        export_order = ('username', 'first_name', 'last_name', 'email',
+                        'is_active', 'date_joined', 'mobile_number',
+                        'alias', 'date_of_birth', 'gender', 'site',
+                        'migrated_username')
+        import_id_fields = ['username']
+        skip_unchanged = True
+
+    def dehydrate_migrated_username(self, user):
+        return user.username
+
+    def dehydrate_security_question_answers(self, user):
+        try:
+            return [(x.question.title, x.answer)
+                    for x in user.profile.securityanswer_set.all()]
+        except:
+            return None
+
+    def export(self, queryset=None, *args, **kwargs):
+        qs = self._meta.model.objects.exclude(
+            Q(is_staff=True) | Q(is_superuser=True))
+        return super(MergedCMSUserResource, self).export(
+            qs, *args, **kwargs)
+
+    def get_prefixed_username(self, data):
+        return '%s_%s' % (
```

Comment:
Please can we use `.format` here instead of `%s` (for ref: https://www.python.org/dev/peps/pep-3101/ )